simulation_runner/tests/integration/test_bootstrap_integration.py

```
@@ -0,0 +1,218 @@
+        os.getenv("ENVIRONMENT") not in ["staging", "production"],
+        reason="AWS Parameter Store only available in staging/production",
+    )
+    def test_config_loaded_from_aws(self):
+        """Test that configuration loads from AWS Parameter Store in production.
+
+        This test only runs in CI/staging/production environments where
+        AWS Parameter Store is available.
+        """
+        # Build CLI
+        subprocess.run(
+            ["pants", "package", "simulation_runner:simulation-runner-cli"],
```

Ruff (0.14.3)

Starting a process with a partial executable path

(S607)